### PR TITLE
Remove text and location section from homepage

### PR DESCRIPTION
**Pull request by SteveChenDC**
Since text will be on other pages and location section will be it's own page. Generally makes it alot less wordy.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1077,24 +1078,6 @@ <h4 class="pricing-col-header-amount"><span class="pricing-col-amount">$200</spa
...
-          <li itemprop="address"><i class="fa fa-map-marker highlight"></i>1500 Remount Rd, Front Royal, VA 22630</li>
-          <li itemprop="email"><i class="fa fa-envelope highlight"></i>info@rubyforgood.org</li>
-          <li itemscope="" itemtype="http://schema.org/Event"><i class="fa fa-clock-o highlight"></i>
-          <time itemprop="startDate" datetime="2016-06-16T07:30">3:00PM 6/16</time> - <time itemprop="endDate" datetime="2016-06-19T22:00">5:00PM 6/19</time>
-          </li>
-        </ul>
-      </div>
-    </div>
-  </div>
-</section>
```
Comment (csexton) [inline index.html:1097]: I think we want to keep this section on the homepage. It looks nice and does keep the address on the initial page.

**Comment by csexton** [comment]
The copy updates are great. 💯

**Comment by SteveChenDC** [comment]
Added map section back in. 🎯

File: index.html (hunk 1 above)
Comment (adunkman) [inline index.html:1097]: [i’m the map!](https://www.youtube.com/watch?v=VdQcdfd0Fec)

**Comment by kalimar** [comment]
gif-keyboard-2733439688708356239

**Comment by adunkman** [comment]
Looks good to me! 👍 

hamion

**Comment by csexton** [comment]
LGTM

whew

**Pull request merged**